### ISSUE TITLE
Support lineary and exponentially chirped beeps

### ISSUE THREAD

**Issue by KOLANICH** [spkr-beep/beep#14]
`-C (l|e) <first freq> <last freq>`

**Comment by ndim** [comment]
How would you like that to work? We can only change the frequency of a square wave here.

I am not even sure whether changing the frequency starts a new square right now or only when the next wave period starts.

**Comment by KOLANICH** [comment]
Absolutely. Currently I just use a sequence of beeps changing their freq, but I don't think the result is optimal. So, it is proposed to do the following:
1. implement dumb "chirps" just sounding somehow like chirps, using just a sequence of beeps
2. try to find a way to implement real chirps without usimg snd_pcsp (BTW, I tried to output to it from Audacity with zero success)

**Comment by ndim** [comment]
Looks to me like using (a variant of) sox' `play -n synth 0.25 sine 100-5000 gain -10` would better do what you are describing.

**Comment by KOLANICH** [comment]
Does it use PC Speaker?

**Comment by ndim** [comment]
No. sox uses normal PCM sound via sound card.

**Comment by KOLANICH** [comment]
So, it is not a complete replacement to `beep`.

**Comment by ndim** [comment]
Certainly not. But that does produce a proper chirp, AFAICT without a digital storage oscilloscope.
